core/src/main/groovy/noe/rhel/server/workspace/WorkspaceHttpdTomcatsBaseOS.groovy

```diff
@@ -0,0 +1,84 @@
+package noe.rhel.server.workspace
+
+import groovy.util.logging.Slf4j
+import noe.common.DefaultProperties
+import noe.common.utils.FileStateVault
+import noe.common.utils.JBFile
+import noe.common.utils.Version
+import noe.ews.server.tomcat.TomcatProperties
+import noe.rhel.server.httpd.HttpdBaseOS
+import noe.server.Httpd
+import noe.workspace.WorkspaceHttpdBaseOS
+import noe.workspace.WorkspaceMultipleTomcats
+import noe.workspace.WorkspaceTomcat
+
+@Slf4j
+class WorkspaceHttpdTomcatsBaseOS extends WorkspaceMultipleTomcats{
+
+  int numberOfAdditionalTomcats
+
+  protected static WorkspaceHttpdBaseOS workspaceHttpd
+  protected static WorkspaceTomcat workspaceTomcat
+  private FileStateVault moduleJkStateVault
+
+  Boolean ews
+  Boolean rpm
+  Version ewsVersion
+  String hostIpAddress = DefaultProperties.HOST
+  Map<String,String> originalTomcatHosts = [:]
+
+  WorkspaceHttpdTomcatsBaseOS(int numberOfAdditionalTomcats = 0) {
+    super()
+    this.numberOfAdditionalTomcats = numberOfAdditionalTomcats
+    //downloadClusterBench()
+    workspaceHttpd = new WorkspaceHttpdBaseOS()
+    workspaceTomcat = new WorkspaceTomcat()
+
+    this.ewsVersion = DefaultProperties.ewsVersion()
+  }
+
+  def prepare() {
+    log.debug("Creating workspace Httpd with ${numberOfAdditionalTomcats} Tomcats BaseOS.")
+    super.prepare()
+    workspaceHttpd.prepare()
+
+    HttpdBaseOS httpd = (HttpdBaseOS) serverController.getServerById(serverController.getHttpdServerId())
+    httpd.setHost(hostIpAddress)
+    httpd.updateConfSetBindAddress(hostIpAddress)
+
+    copyModulesIfMissing(httpd)
+    workspaceTomcat.prepare(true) //always true due to running with JWS 5 `ewsVersion >= new Version('3.1.0-DR0')`
+
+    createAdditionalTomcatsWithRegistrations(numberOfAdditionalTomcats)
+    originalTomcatHosts.putAll(originalTomcatHostsIpAddresses())
+
+    serverController.backup()
```

Comment:
Is this OK? 

You do back up after you changed some configurations and you overriding already existing backup of clean installation performed by parents.

Reply:
I've been nervous about it for quite a some time. I am not 100% sure what should be the right thing as multiple workspaces are implemented it in different ways. I am suspicious about preserving workspace specific set up during Test Suite execution.

Reply:
So what do you suggest to do to be sure this approach is correct?

Reply:
I've tested noe-core without the `backup()` in the end of method and everything seems OK. So removing from code. Updated, squashed.